plugins/generator-1.18.2/forge-1.18.2/livingentity.definition.yaml

```diff
@@ -1,3 +1,8 @@
+global_templates:
+  - template: elementinits/breedableentities.java.ftl

```

Comment:
I'm pretty sure it be done inside the renderer class

Reply:
it requires less code with the event

Reply:
I'd move the new method to one of existing classes generated from common entity global templates (the ones defined in `common.entities.yaml`, they generate for all mod elements adding custom entities) or from regular living entity templates.
I would recommend doing the same for shields property functions registration on their pull request, but that one would then conflict with item properties PR due to modifying the same part of the same file.